node/src/node_configurator/mod.rs

```diff
@@ -240,18 +421,27 @@ mod tests {
         );
         std::env::set_var("MASQ_CONFIG_FILE", "booga.toml");
 
-        let (config_file_path, user_specified, _data_dir, _real_user) = determine_fundamentals(
+        let user_specific_data = determine_user_specific_data(
             &DirsWrapperReal {},
             &determine_config_file_path_app(),
             args_vec.as_slice(),
         )
         .unwrap();
         assert_eq!(
-            format!("{}", config_file_path.parent().unwrap().display()),
-            data_directory.to_string_lossy().to_string(),
+            format!(
+                "{}",
+                user_specific_data.config_file.parent().unwrap().display()
+            ),
+            user_specific_data
+                .data_directory
+                .to_string_lossy()
+                .to_string(),
+        );
+        assert_eq!(
+            "booga.toml",
+            user_specific_data.config_file.file_name().unwrap()
         );
-        assert_eq!("booga.toml", config_file_path.file_name().unwrap());
-        assert_eq!(true, user_specified);
```

Comment:
Can you add an assert to preserve this old proof that the config file is marked as being user-specified?